coldfront/plugins/qumulo/utils/eib_billing.py

```diff
@@ -0,0 +1,187 @@
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
+
+YYYY_MM_DD = "%Y-%m-%d"
+
+REPORT_HEADER = """Submit Internal Service Delivery,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Area,All,,Business Process Parameters,Internal Service Delivery Data,,,,,,,Internal Service Delivery Line Data+,,,,,,,,,,,,,
+Restrictions,Required,Optional,Optional,Optional,Optional,Required,Required,Required,Required,Optional,Required,Optional,Required,Optional,Optional,Optional,Optional,Optional,Required,Optional,Optional,Optional,Optional. May have multiples,Optional. May have multiples
+Format,Text,Y/N,Y/N,Text,Y/N,Company_Reference_ID,Internal_Service_Provider_ID,Currency_ID,YYYY-MM-DD,Text,Text,Text,Number,Text,Spend_Category_ID,Number (22,2),UN_CEFACT_Common_Code_ID,Number (26,6),Number (18,3),Employee_ID,YYYY-MM-DD,Text,Cost_Center_Reference_ID,Fund_ID
+Fields,Spreadsheet Key*,Add Only,Auto Complete,Internal Service Delivery ID,Submit,Company*,Internal Service Provider*,Currency*,Document Date*,Memo,Row ID**,Internal Service Delivery Line ID,Internal Service Delivery Line Number*,Item Description,Spend Category,Quantity,Unit of Measure,Unit Cost,Extended Amount*,Requester,Delivery Date,Memo,Cost Center,Fund,,,,USAGE,RATE,UNIT
+"""
+
+QUERY_MONTHLY_BILLING = """
+SELECT
+  NULL fields,
+  ROW_NUMBER() OVER (ORDER BY report.service_name) spreadsheet_key,
+  'N' add_only,
+  'N' auto_complete,
+  NULL internal_service_delivery_id,
+  'Y' submit,
+  'CP0001' company,
+  'ISP0000030' internal_service_provider,
+  'USD' currency,
+  '%s' document_date,
+  'FY25 ' || '%s' || ' Monthly for ' || report.sponsor memo,
+  '1' row_id,
+  NULL internal_service_delivery_line_id,
+  '1' internal_service_delivery_line_number,
+  '"' || 'WashU IT RIS ' || report.service_name || ' - ' || report.service_rate_category || '; Usage: ' || report.billing_unit || ' X Rate: ' || report.rate || ' X Per: ' || report.service_unit || '"' item_description,
+  'SC510' spend_category,
+  '1' quantity,
+  'EA' unit_of_measure,
+  report.fee unit_cost,
+  report.fee extended_amount,
+  NULL requester,
+  report.delivery_date delivery_date,
+  '"' || report.storage_name || '"' fileset_memo,
+  report.cost_center cost_center,
+  NULL fund,
+  NULL,
+  NULL,
+  NULL,
+  report.billing_unit usage,
+  report.rate rate,
+  report.service_unit unit
+FROM (
+  SELECT
+    '%s' delivery_date,
+    data.service_unit,
+    data.storage_name,
+    data.service_name,
+    data.sponsor,
+    data.billing_unit,
+    data.rate,
+    data.service_rate_category,
+    data.department_number,
+    data.cost_center,
+    data.billing_unit*data.rate fee
+  FROM (
+    SELECT
+      DATE(most_recent_haau.recent_modified) usage_date,
+      '1' service_id,
+      department_number,
+      storage_name,
+      'Storage2 Active' service_name,
+      u.username sponsor,
+      service_rate_category,
+      cost_center,
+      'monthly' billing_cycle,
+      true subsidized,
+      false exampt,
```

Comment:
Typo - `exampt` -> `exempt`.

Just bringing it up in case it's new - if it's already part of the report format used in Box, that's probably fine to leave.

Reply:
It's a typo. Thanks!